module_auto_update/data/cron_data.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Tecnativa - Jairo Llopis
```

Comment:
Can you add the LasLabs copyright here too? Not sure why it's not there, but I'm pretty sure this file came from my side originally